src/components/alertingTransformations.ts

```diff
@@ -13,8 +13,8 @@ export const labelToProm = (labels?: Label[]) => {
 export const transformAlertFormValues = (alertValues: AlertFormValues | undefined): AlertRule => {
   return {
     alert: alertValues?.name ?? '',
-    expr: `${ALERT_PROBE_SUCCESS_RECORDING_METRIC}{alert_sensitivity="${alertValues?.sensitivity?.value}"} < ${
-      alertValues?.probePercentage ?? ''
+    expr: `${alertValues?.metric}{alert_sensitivity="${alertValues?.sensitivity?.value}"} ${alertValues?.operator} ${
```

Comment:
`metric` is not part of the editable values, but I need to access the metric name here...